java-sockets/src/test/java/com/baeldung/threading/ThreadModelManualTest.java

```diff
@@ -0,0 +1,45 @@
+package com.baeldung.threading;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.PrintWriter;
+import java.net.Socket;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class ThreadModelManualTest {
```

Comment:
Let's add a comment that the test assumes that ThreadPerConnectionServer or ThreadPerRequestServer has been started already externally